lib/kvdb/kvdb_cparams.c

```diff
@@ -264,8 +264,22 @@ kvdb_cparams_defaults()
     return params;
 }
 
+static void
+kvdb_storage_defaults_set(struct kvdb_cparams *params, bool pmem_only)
```

Comment:
In my opinion, I don't think this function adds too much, and inlining it would probably be better.